packages/long-short-pair/README.md

```diff
@@ -0,0 +1,7 @@
+# Optimistic Oracle Subgraph
+
+## Deploying the subgraph
+
+- Mainnet: `yarn prepare:mainnet && yarn deploy:mainnet`
+- Kovan: `yarn prepare:kovan && yarn deploy:kovan`
+- Polygon: `yarn prepare:polygon && yarn deploy:polygon`
```

Comment:
Is this going to be deployed on polygon?

Reply:
yup! we have LSP contracts on polygon so no reason to not include the subgraph.